server/src/task_set.ts

```diff
@@ -64,13 +69,10 @@ export class TaskSet extends EventEmitter<{
     if (serialization.model.isEncoded(model)) {
       encodedModel = model // don't do anything if already encoded
     } else {
-      let tfModel: Model
+      let tfModel: Model<DataType>
       if (model === undefined) { 
         // Get the model if nothing is provided
         tfModel = await this.loadModelFromTask(taskOrProvider)
-      } else if (model instanceof URL) {
-        // Downloading the model if a URL is given
-        tfModel = new models.TFJS(await tf.loadLayersModel(model.href))
```

Comment:
yeah, we don't have a model URL in `Task` anymore, it was move in `getModel` some time back (and in fact, one can quick easily have the old behavior by making a `TaskProvider` where it will `tf.loadLayersModel`)